src/Resources/config/services.yml

```diff
@@ -1,4 +1,16 @@
 services:
+    contao.controller.abstract_csv_import:
+        class: Contao\CoreBundle\Controller\AbstractCsvImportController
+        abstract: true
+        calls:
```

Comment:
As these are mandatory, you must use constructor injection rather than setter injections. Setter injections are for optional dependencies.